src/app/views/query-runner/request/feedback/uiStrings.ts

```diff
@@ -6,7 +6,8 @@ export const uiStringMap: any = {
   Prompt_YesLabel: translateMessage('Sure'),
   Prompt_NoLabel: translateMessage('Sure'),
   Graph_Explorer_Rating_Question: translateMessage('Graph Rating Question'),
-  Rating_Values_1: `1 - ${translateMessage('Not at all likely')}`,
```

Comment:
I know I had suggested removing the strings, this was because of the wrong interpretation on the OCV side. Now that everything is working as expected I think you can just return them
(i.e 0 - not at all likely and 10 - Extremely likely)

Reply:
Awesome. Done